### PR TITLE
(#53) Pager Should Not Appear if results are less than page unit

### LINKED ISSUE #53: Pager Should Not Appear if ≤ 20 Results (can resolve post launch)
**Note:** This issue is not a deal breaker for launch

### Issue description
If there are 20 or less search results, the pager should not appear since there are no additional pages for the user to toggle through. This is the case with CTS results. But SWS shows the pager (http://ncigovcdode377.prod.acquia-sites.com/search/results?swKeyword=mynci)

> **ESTIMATE**

### Steps to reproduce the issue

1.  On CGov, search `mynci`
2. You should get less than 20 results back (currently 8 results)  
3. Top and bottom pager appears on search results page, showing only 1 page

### What's the expected result?
- No pager should appear (see CTS example with only a few results - no pager appears: https://www.cancer.gov/about-cancer/treatment/clinical-trials/search/r?loc=0&rl=2&stg=C8815&t=C3158)

### What's the actual result?
- Top and bottom pager appears on search results page, showing only 1 page

### Additional details / screenshot
image (1)

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #53 

* Fixing pager to not display if returned results are less than page units

### CODE REVIEW

File: src/components/molecules/search-results-list/search-results-list.jsx [hunk 1]
```diff
@@ -93,14 +96,16 @@ const SearchResultsList = ({
...
 				</h4>
-				<SearchResultsPager
-					testid={testIds.RESULTS_PAGER_TOP}
-					current={currentPage}
-					totalResults={results.totalResults}
-					resultsPerPage={resultsPerPage}
-					language={language}
-					keyword={keyword}
-				/>
+				{!hidePager && (
```
Comment (arcepaul) [inline src/components/molecules/search-results-list/search-results-list.jsx:99]: can we reverse the logic on this to make it a positive case?  like `showPager &&`

File: src/components/molecules/search-results-list/search-results-list.scss [hunk 2]
```diff
@@ -60,7 +61,7 @@
 	appearance: none;
 	background-color: white;
 	background-position: right 12px center;
-	background-size: 8px;
+	background-size: 0.5rem;
```
Comment (arcepaul) [inline src/components/molecules/search-results-list/search-results-list.scss:64]: rem or em?  I think we're using em everywhere else(for the time being).

Comment (karlikpj) [reply src/components/molecules/search-results-list/search-results-list.scss:64]: Ooof - no was left from trying to fix IE - will remove!

File: src/components/molecules/search-results-list/search-results-list.jsx [hunk 3]
```diff
@@ -84,7 +84,10 @@ const SearchResultsList = ({
 			</Helmet>
 		);
 	};
-
+	console.log(results.result.map);
```
Comment (kate-mashkina) [inline src/components/molecules/search-results-list/search-results-list.jsx:87]: console.log() should probably go... :)

**Pull request closed without merging**